include/constants.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include <string>
+
+namespace diskann {
+
+ class Constants {
+   public:
+    const static std::string starting_points_data_file_suffix;
+    const static std::string starting_points_id_file_suffix;
```

Comment:
In this class, some members seem like parameter names and some seem like parameter values. Would it be better to encapsulate the parameter values in an enum? Also, do you think there is a need for a sepearate header file with this class alone?

Reply:
just want to have a place to store the common constant strings. because they may be used in multiple places, and want to avoid any miss-spells.